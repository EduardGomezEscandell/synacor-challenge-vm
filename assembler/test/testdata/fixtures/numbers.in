0
16
0x15
0b011100
0x
0b
00