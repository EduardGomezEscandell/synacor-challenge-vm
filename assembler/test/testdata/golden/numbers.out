<NUMBER 0> <EOL>
<NUMBER 16> <EOL>
<NUMBER 21> <EOL>
<NUMBER 28> <EOL>
<ERROR Unexpected end of digit in base-16 number literal>
<ERROR Unexpected end of digit in base-2 number literal>
<NUMBER 0> <EOL>
